chainer/training/extensions/parameter_statistics.py

```diff
@@ -52,17 +79,9 @@ class ParameterStatistics(extension.Extension):
     report_key_template = ('{prefix}{link_name}{param_name}/{attr_name}/'
                            '{function_name}')
 
-    default_statistics = {
-        'mean': lambda x: backend.get_array_module(x).mean(x),
-        'std': lambda x: backend.get_array_module(x).std(x),
-        'min': lambda x: backend.get_array_module(x).min(x),
-        'max': lambda x: backend.get_array_module(x).max(x),
-        'zeros': lambda x: backend.get_array_module(x).count_nonzero(x == 0),
-        'percentile': lambda x: backend.get_array_module(x).percentile(
-            x, (0.13, 2.28, 15.87, 50, 84.13, 97.72, 99.87))
-    }
-
-    def __init__(self, links, statistics=default_statistics,
+    default_statistics = _default_statistics
```

Comment:
Is this attribute needed?

Reply:
I think its convenient for users.
Let's discuss the topic in the separate issue if needed.

Reply:
OK.